conf/dependency-versions-config.json

```diff
@@ -15,7 +15,7 @@
     { "group": "uk.gov.hmrc", "name": "sbt-utils" }
   ],
   "libraries": [
-    { "group": "org.reactivemongo", "name": "reactivemongo", "latestVersion": "0.20.3" },
+    { "group": "org.reactivemongo", "name": "reactivemongo", "latestVersion": "0.18.8" },
```

Comment:
Also, BDOG-657 suggests that we still want people on 0.18.8 as 0.20.3 had breaking changes to get there.